requirements/prod.txt

```diff
@@ -38,37 +38,18 @@ versiontools==1.9.1
 cef==0.3
 pyparsing==1.5.4
 
-# from funfactory
-
-# Django stuff
--e git://github.com/jbalogh/django-multidb-router.git#egg=django-multidb-router
--e git://github.com/jsocol/django-cronjobs.git#egg=django-cronjobs
+###################
+# from funfactory #
+###################
 
 # Forms
--e git://github.com/mozilla/happyforms.git#egg=happyforms
-
-# Templates
--e git://github.com/jbalogh/jingo.git#egg=jingo
--e git://github.com/jsocol/jingo-minify.git#egg=jingo-minify
-
-# Various tidbits
--e git://github.com/jsocol/commonware.git#egg=commonware
--e git://github.com/mozilla/nuggets.git#egg=nuggets
-
-# Security
--e git://github.com/fwenzel/django-sha2.git#egg=django-sha2
--e git://github.com/jsocol/bleach.git#egg=bleach
--e git://github.com/mozilla/django-session-csrf.git#egg=django-session-csrf
+git+git://github.com/mozilla/happyforms.git#egg=happyforms
```

Comment:
Are we even really using happyforms? It doesn't look like Cyder ignores leading whitespace on my end.